packages/web/src/components/track/GiantTrackTile.tsx

```diff
@@ -34,6 +34,7 @@ import moment from 'moment'
 import { useDispatch, shallowEqual, useSelector } from 'react-redux'
 
 import IconRobot from 'assets/img/robot.svg'
+import DownloadButtons from 'components/download-buttons/DownloadButtons'
```

Comment:
just double checking this was the result of running `npm run verify` ?

Reply:
yeah seems so